### DISCUSSION TITLE
Source Code for Demos

### DISCUSSION THREAD

**Discussion by mendy-friedman** [Comcast/react-data-grid#3095] [General]
I'm new to react-data-grid (and fairly new to React) and I'd love to play around with the examples showcased on the [Demo](https://adazzle.github.io/react-data-grid/#/all-features) page. Are these examples available somewhere in Code Sandbox or similar where I can explore the code and play around? Or is the source code for the examples available, even if it's not in a live sandbox?

Curious how others have learned to use this great package.

**Comment by nstepien** [comment]
The code for the examples are all here: https://github.com/adazzle/react-data-grid/tree/main/website/demos

**Reply by mendy-friedman** [reply]
Ah i see - this is great. Thank you!

**Reply by paustint** [reply]
@mendy-friedman - Also, if you were not aware - once you navigate to any repo on github and are looking at the code, you can press the period `.` key on your keyboard and that will open up the codebase in the online web version of vscode which is a great way to explore the examples.

**Reply by mendy-friedman** [reply]
This just blew my mind... I did not know that and this is awesome. Thank you!
